core/shared/src/main/scala/fs2/Stream.scala

```diff
@@ -4117,6 +4114,21 @@ object Stream extends StreamLowPriority {
       parJoin(Int.MaxValue)
   }
 
+  /** Provides syntax for a stream of `F` effects. */
+  implicit class StreamFOps[F[_], O](private val self: Stream[F, F[O]]) {
```

Comment:
good to know @Daenyth, I'd never think it could blow the memory. Would a suspend help resolve that? (at least to reduce the overhead of the non-trivial ones)

Reply:
Probably not because the issue is the memory used by having large `Chunk[IO[A]]`

It's also possibly my codebase just was doing something really silly and that's why it cost so much memory for that structure